src/sentry/models/savedsearch.py

```diff
@@ -7,17 +7,6 @@
 from sentry.models.search_common import SearchType
 
 
-DEFAULT_SAVED_SEARCHES = [
-    {"name": "Unresolved Issues", "query": "is:unresolved", "is_default": True},
-    {"name": "Needs Triage", "query": "is:unresolved is:unassigned"},
-    {"name": "Assigned To Me", "query": "is:unresolved assigned:me"},
-    {"name": "My Bookmarks", "query": "is:unresolved bookmarks:me"},
-    {"name": "New Today", "query": "is:unresolved age:-24h"},
-    {"name": "Unhandled Errors", "query": "is:unresolved error.unhandled:true"},
-]
-DEFAULT_SAVED_SEARCH_QUERIES = set(search["query"] for search in DEFAULT_SAVED_SEARCHES)
-
-
 class SavedSearch(Model):
```

Comment:
I think we can also remove the `project` column and any related rows now that this is no longer used. Same for `is_default`, it looks like both those cols were only used for sentry 9

Reply:
I was going to remove those columns once this had merged.